### PR TITLE
emailing module now functioning

### PR DESCRIPTION
**Pull request by adrianmc15**
- set up a backend emailing service
- integrated with the existing code

### CODE REVIEW

**Review by gang21 — APPROVED** [review]

**Comment by gang21** [comment]
looks great! are you able to merge still even if Vercel is failing?

File: backend/src/utils/mailer.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import nodemailer from 'nodemailer';
+
+async function sendEmail(to: string, subject: string, text: string) {
+    // Create a transporter object using Gmail SMTP transport
+    let transporter = nodemailer.createTransport({
```
Comment (ribru17) [inline backend/src/utils/mailer.ts:5]: Prefer:
```suggestion
    const transporter = nodemailer.createTransport({
```

File: backend/src/utils/mailer.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        service: 'gmail',
+        auth: {
+            user: 'fraud.ninja.team@gmail.com', // Replace with your Gmail address
+            pass: 'nosq zgue eloq mzaj', // Replace with your App Password or Gmail password
+        },
+    });
+    // Send mail with defined transport object
+    let info = await transporter.sendMail({
```
Comment (ribru17) [inline backend/src/utils/mailer.ts:13]:
```suggestion
    const info = await transporter.sendMail({
```

**Comment by ribru17** [comment]
We should squash that last commit so the email never shows up in the history in the first place:
```bash
git rebase -i HEAD~2
#  change :
# pick a471b99 one small change
#  to :
# s a471b99 one small change
# then :
# quit the buffer
git push -f
```

**Comment by ribru17** [comment]
Also needs prettier formatting please

File: backend/src/utils/mailer.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import nodemailer from 'nodemailer';
+
+async function sendEmail(to: string, subject: string, text: string) {
+  // Create a transporter object using Gmail SMTP transport
+  let transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: 'fraud.ninja.team@gmail.com', // Replace with your Gmail address
+      pass: 'nosq zgue eloq mzaj', // Replace with your App Password or Gmail password
```
Comment (ribru17) [inline backend/src/utils/mailer.ts:9]: Should `pass` be read from `.env`? Also I'm assuming these comments should be removed

**Pull request merged**